app/generators/new_service_generator.rb

```diff
@@ -17,5 +17,24 @@ def to_metadata
       metadata['service_name'] = service_name
       metadata['created_by'] = current_user.id
     end
+
+    metadata['standalone_pages'] = footer_pages
+    metadata
+  end
+
+  private
+
+  def footer_pages
+    I18n.t('footer').map do |attributes|
+      metadata = NewPageGenerator.new(
+        page_type: 'standalone',
+        page_url: attributes[:url]
+      ).to_metadata
```

Comment:
Do the standalone pages need an UUID?

Reply:
So with my changes from main, things would change on the UUID that can lead to a bug here. Now we need to pass the UUID to the new page generator.

Reply:
ah yes. forgot about that. will update